drivers/clock_control/clock_control_r8a779f0_cpg_mssr.c

```diff
@@ -116,22 +162,88 @@ int r8a779f0_cpg_mssr_start_stop(const struct device *dev, clock_control_subsys_
 
 static uint32_t r8a779f0_get_div_helper(uint32_t reg_val, uint32_t module)
 {
+	uint32_t divider = RCAR_CPG_NONE;
+
 	switch (module) {
```

Comment:
Please return divider values directly in cases to meet current version of `r8a7795` clock driver

Reply:
fixed